crates/concrete_check/src/lib.rs

```diff
@@ -222,5 +222,24 @@ pub fn lowering_error_to_report(
                 .with_message(msg)
                 .finish()
         }
+        LoweringError::CallParamCountMismatch {
+            span,
+            found,
+            needs,
+            program_id,
+        } => {
+            let path = file_paths[program_id].to_str().unwrap().to_string();
```

Comment:
would using [diaply](https://doc.rust-lang.org/std/path/struct.Path.html#method.display) instead of converting to str and then String be better? i.e. `file_paths[program_id].display()`